### DISCUSSION TITLE
Cloud Director

### DISCUSSION THREAD

**Discussion by UAnton** [keycloak/keycloak#20374] [Q&A]
Hi,
Can I use Keycloak for SSO (SAML) for VMware Cloud Director? I have a problem. I create a client (vCD) and a User but Cloud Director uses a different URL for each organization and if I create a client for the second user, the first user sees the application link too. Users can see all applications. How can I create 1 user = 1 application?  Thx!

**Comment by danie-dejager** [comment]
Interesting question. Came here as I am wondering about the same problem. Maybe update the question's subject and add SAML or SSO?
